terminus/builders/procedural_city/vertex_graph_to_roads_converter.py

```diff
@@ -18,23 +18,29 @@ def get_roads(self):
                                  key=lambda vertex: len(vertex.neighbours))
             vertex = to_traverse.pop(0)
             while vertex.neighbours:
-                street = Street()
-                street.width = 5
-                street.add_point(vertex.coords)
-                self._build_road(street, None, vertex)
-                roads.append(street)
+                if vertex.minor_road:
+                    road = Street(width=4)
```

Comment:
Just curious about the 4 and 22 values. Was 5 too wide? Also, I would sort of expect the trunk to be 4 times the street (or something along those lines), as it has 4 lanes. What rationale you used for 22?